### PR TITLE
[controller] Update parent colo version status on killjob

### PR DESCRIPTION
**Pull request by majisourav99**
## Update parent colo version status on killjob

The nuage page shows the version status from parent colo. For killed pushes, the status was not updated on the nuage leading users to think it was successful. This PR update the status for killjob on parent colo. Also fixes list_backup_version admin path.

## How was this PR tested?

CI
## Does this PR introduce any user-facing changes?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure to explain your proposed changes and call out the behavior change.

### CODE REVIEW

File: services/venice-controller/src/main/java/com/linkedin/venice/controller/server/AdminSparkServer.java [hunk 1]
```diff
@@ -313,7 +313,7 @@ public boolean startInner() throws Exception {
     httpService.get(CLUSTER_HEALTH_STORES.getPath(), storesRoutes.getAllStoresStatuses(admin));
     httpService.get(STORE.getPath(), storesRoutes.getStore(admin));
     httpService.get(FUTURE_VERSION.getPath(), storesRoutes.getFutureVersion(admin));
-    httpService.get(BACKUP_VERSION.getPath(), storesRoutes.getFutureVersion(admin));
+    httpService.get(BACKUP_VERSION.getPath(), storesRoutes.getBackupVersion(admin));
```
Comment (huangminchn) [inline services/venice-controller/src/main/java/com/linkedin/venice/controller/server/AdminSparkServer.java:316]: Do we have a test for it?

Comment (majisourav99) [reply services/venice-controller/src/main/java/com/linkedin/venice/controller/server/AdminSparkServer.java:316]: yes, I added the test in TestHybrid.java

Comment (huangminchn) [reply services/venice-controller/src/main/java/com/linkedin/venice/controller/server/AdminSparkServer.java:316]: Thanks!

**Review by huangminchn — APPROVED** [review]
Thanks a lot Sourav!

**Pull request merged**